graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -128,7 +128,7 @@ abstract class VertexRDD[VD](
    *
    * @param other the other RDD[(VertexId, VD)] with which to diff against.
    */
-  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD]
+  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD] = ???
```

Comment:
BTW let me use the same fix that Sean mentioned. That will tighten the exclusion so that other methods are still checked by MiMA.